libs/state/src/lib/rx-state.service.ts

```diff
@@ -547,10 +583,24 @@ export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
    * @internal
    */
   select<R>(
```

Comment:
is there a reason why we just don't re-use `select` here? Just came to my mind

Reply:
You mean why doesn't the `RxState#select` method just apply the `select` operator with the same arguments? That's a very good question, I don't know why the implementations are duplicated :smile: Was there some reason for it?

Reply:
that's what i'm asking myself ^^ is it possible to switch it? It's maybe the case because it used lived in the cdk package

Reply:
OK, I've refactored it :heavy_check_mark:

Reply:
great job!!!